### DISCUSSION TITLE
Question about acquisition rate regarding OASIS

### DISCUSSION THREAD

**Discussion by smbaca** [flatironinstitute/CaImAn#1352] [Q&A]
We normally acquire at 20Hz and another caiman user recommended a higher frame rate. On the flip side, the Inscopix folks recommend no more than 10 Hz with their pipeline with CNMFE:  https://github.com/inscopix/inscopix-cnmfe 

"Temporal downsampling of the data to 10 Hz or below. On each iteration of the algorithm, temporal traces are deconvolved using an autoregressive model of order 1. Higher-frequency data may not be adequately deconvolved using low-order models in noisy regimes. The current version of Inscopix CNMF-E performs deconvolution using the OASIS (https://pubmed.ncbi.nlm.nih.gov/28291787/) algorithm with an AR(1) model, which is appropriate for data recorded at up to 10 Hz."

The other question is that exposure and acquisition can be decoupled. I can integrate over 100ms duration for a 10Hz fps rate, or I can also go down to around 5ms (adjusting light intensity of course). Are there are any best practices or suggestions from the 1P people?

**Comment by bioBaddie** [comment]
It all depends on the kinetics of the calcium indicator you are using. Which one are you using?

**Comment by smbaca** [comment]
Gcamp6fSent from my iPhoneOn May 16, 2024, at 11:35 AM, Alexandra Carrizales ***@***.***> wrote:﻿
It all depends on the kinetics of the calcium indicator you are using. Which one are you using?

**Reply by bioBaddie** [reply]
For GCaMP6f I have seen others image at 20 Hz or higher. With a fast indicators like GCaMP6f I imagine the rise time is nearly instantaneous so I would recommend setting the AR model to 1. Determining appropriate imaging parameters is also dependent on your microscope setup. I would recommend doing a small pilot study to figure out what works best for you to avoid phototoxicity. My lab recently did a similar pilot study because we noticed our cells were behaving strangely after repeated recordings. Turns out we had the laser power up too high. My lab uses GCamP6s, which is much slower, so we only image at 10 Hz with 100ms exposure (1p imaging). With slower indicators the rise time isn't instantaneous so I set the AR model to 2. I hope this helps!

**Comment by smbaca** [comment]
Thanks for the response. We have been going at 20 Hz and 30 Hz but then the Inscopix CNMFe protocol suggested that maybe we should be going slower. We have been using the AR parameter set to one, but I will check the other parameters. If you are willing to share, I’m curious how long you record for a single session and the degree of photobleaching that you get.Sent from my iPhoneOn May 16, 2024, at 12:08 PM, Alexandra Carrizales ***@***.***> wrote:﻿
For GCaMP6f I have seen others image at 20 Hz or higher. With a fast indicators like GCaMP6f I imagine the rise time is nearly instantaneous so I would recommend setting the AR model to 1. Determining appropriate imaging parameters is also dependent on your microscope setup. I would recommend doing a small pilot study to figure out what works best for you to avoid phototoxicity. My lab recently did a similar pilot study because we noticed our cells were behaving strangling after repeated recordings. Turns out we had the laser power up to high. My lab uses GCamP6s, which is much slower, so we only image at 10 Hz with 100ms exposure (1p imaging). With slower indicators the rise time isn't instantaneous so I set the AR model to 2. I hope this helps!

**Reply by bioBaddie** [reply]
Photobleaching used to be a problem for us, but since we experimented with our imaging parameters that problem has been fixed. The biggest change we made was reducing the laser power. Haven't experimented with super long imaging sessions, but have been collecting 3 videos that are each 5-min long (for a single plate) and have not noticed any photobleaching effects. Prior to adjusting the imaging parameters I was noticing significant photobleaching by the 2nd recording. Best of luck to you!
